src/components/MapView/Legends/index.tsx

```diff
@@ -200,12 +270,44 @@ function LegendItem({
               <Typography variant="h5">{children}</Typography>
             </Grid>
           )}
+
+          {exposure && !analysisResult ? (
+            <AnalysisButton
+              variant="contained"
+              color="primary"
+              size="small"
+              onClick={runExposureAnalysis}
+            >
+              exposure analysis
+            </AnalysisButton>
+          ) : null}
```

Comment:
You don't need a ternary with null, just do `exposure && !analysisResult && <component/>`

Reply:
Included in ad9bb6582996ad09efe7f5f096acc9e8ef550a75. I'll resolve the remaining comments.